--- libpng-1.5.28/contrib/pngminus/makefile.std.orig	2016-12-29 01:35:07.000000000 +0100
+++ libpng-1.5.28/contrib/pngminus/makefile.std	2017-02-03 23:08:53.032031871 +0100
@@ -12,21 +12,21 @@
 #PNGLIB = -L$(PNGPATH)/lib -lpng15
 #PNGLIBS = $(PNGPATH)/lib/libpng15.a
 PNGINC = -I../..
-PNGLIB = -L../.. -lpng
+PNGLIB = -L../.. -L../../.libs -lpng15
 PNGLIBS = ../../libpng.a
 
 #ZPATH = /usr/local
 #ZINC = -I$(ZPATH)/include
 #ZLIB = -L$(ZPATH)/lib -lz
 #ZLIBS = $(ZPATH)/lib/libz.a
-ZINC = -I../../../zlib
-ZLIB = -L../../../zlib -lz
-ZLIBS = ../../../zlib/libz.a
+ZINC =
+ZLIB = -lz
+ZLIBS = -lz
 
 CPPFLAGS=$(PNGINC) $(ZINC)
-CFLAGS=
-LDLIBS=$(PNGLIB) $(ZLIB)
-LDLIBSS=$(PNGLIBS) $(ZLIBS)
+CFLAGS=$(OPT_FLAGS)
+LDLIBS=$(LDFLAGS) $(PNGLIB) $(ZLIB)
+LDLIBSS=$(LDFLAGS) $(PNGLIBS) $(ZLIBS)
 C=.c
 O=.o
 L=.a
@@ -34,8 +34,7 @@
 
 # dependencies
 
-#all: png2pnm$(E) pnm2png$(E)
-all: png2pnm$(E) pnm2png$(E) png2pnm-static$(E) pnm2png-static$(E)
+all: png2pnm$(E) pnm2png$(E)
 
 png2pnm$(O): png2pnm$(C)
 	$(CC) -c $(CPPFLAGS) $(CFLAGS) png2pnm$(C)
